identity-service/src/authMiddleware.js

```diff
@@ -72,4 +72,38 @@ async function authMiddleware (req, res, next) {
   }
 }
 
+async function parameterizedAuthMiddleware ({ shouldRespondBadRequest }) {
```

Comment:
small comment would be nice. took me a min to figure out that it's almost the same as authMiddleware but wrapped in a function that takes a param that can optionally to either throw an error or call next

Reply:
added :)